app/pb/api.proto

```diff
@@ -203,22 +203,22 @@ message User {
   double community_standing = 6;
   uint32 num_references = 7;
   string gender = 8;
-  string pronouns = 101;
+  google.protobuf.StringValue pronouns = 101;
   uint32 age = 9;
   google.protobuf.Timestamp joined = 11; // not exact
   google.protobuf.Timestamp last_active = 12; // not exact
   HostingStatus hosting_status = 13;
   MeetupStatus meetup_status = 102;
-  string occupation = 14;
-  string education = 103;
-  string about_me = 15;
-  string my_travels = 104;
-  string things_i_like = 105;
-  string about_place = 16;
+  google.protobuf.StringValue occupation = 14;
+  google.protobuf.StringValue education = 103;
+  google.protobuf.StringValue about_me = 15;
+  google.protobuf.StringValue my_travels = 104;
+  google.protobuf.StringValue things_i_like = 105;
+  google.protobuf.StringValue about_place = 16;
   repeated string languages = 17;
   repeated string countries_visited = 18;
   repeated string countries_lived = 19;
-  string additional_information = 106;
+  google.protobuf.StringValue additional_information = 106;
```

Comment:
This would need a lot of changes in the code because now they have to be accessed with `additional_information.value` and created with `google.protobuf.StringValue(value="")`. Probably one of the other solutions is easier.